### DISCUSSION TITLE
Adding more repositories to "daisyUI example repositories"

### DISCUSSION THREAD

**Discussion by anton-gustafsson** [saadeghi/daisyui#2340] [Q&A]
Hi! I would like to add another framework to the daisyui example repo list. I would love to contribute with a PR for this. How should the stackblitz link be handled. Should a just link the example repo in the PR description, so you @saadeghi can create a stackblitz link?

**Comment by saadeghi** [comment]
Yes. If it's a well-known framework, I would be happy to add.  
Which framework is it?  

If you have a stackblitz link, I can fork it and put the link (It's better if I own the Stackblitz repo so I can update the dependencies when needed)

**Reply by anton-gustafsson** [reply]
Aurelia 2, but I would be fine with Aurelia 1 as well.  I will provide links to the repositories for the framework versions below and also a daisyUI example repository

[Aurelia 2](https://github.com/aurelia/aurelia) - 1.3k stars

[Aurelia 1](https://github.com/aurelia/framework) - 11.8k stars

[Aurelia 2 DaisyUI example](https://github.com/anton-gustafsson/aurelia2-daisyui-example)

Let me know if this is of interest. Then I will create a stackblitz link for you. 

Cheers
